examples/compartmental/lfp.py

```diff
@@ -57,7 +54,7 @@
 lfp.y = [1*mm, 2*mm, 4*mm, 8*mm, 16*mm]
 # Synapses are normally executed after state update, so v-previous_v = dv
 S = Synapses(neuron,lfp,model='''w : ohm*meter**2 (constant) # Weight in the LFP calculation
-                                 v_post = w*(Cm_pre*(v_pre-previous_v_pre)/dt-Im_pre) : volt (summed)''')
+                                 v_post = w*(Ic_pre-Im_pre) : volt (summed)''')
 S.summed_updaters['v_post'].when = 'after_groups'  # otherwise v and previous_v would be identical
```

Comment:
The comment should be changed, since now this is only necessary to avoid the mismatch between `Ic` and `Im` due to a current. Maybe simply: "otherwise Ic has not yet been updated for the current time step."